### PR TITLE
#153728040 Add a help center

### PR DESCRIPTION
**Pull request by Thegaijin**
#### What does this PR do?
Add a help center to the docs

#### Description of Task to be completed?
Adding documentation to help the user find their way around the web app as well as answer questions that may arise regularly.

#### How should this be manually tested?
On loading the page, click on the Docs button on the navigation bar. 
You can check it out [here](https://hc-faqs.herokuapp.com/docs/user-guide/). 

#### The files changed/created include:
Created:
user_guide.html - For the content on the user guide page

Edited
hc/front/urls.py - Added the URL that redirects to the user guide page.
hc/front/views.py - Added the function that loads the page
templates/front/base.html - Add a link to the html page.
templates/front/base_docs.html - Include the link to the users guide page on the docs navigation bar.

#### What are the relevant pivotal tracker stories?
[#153728040](https://www.pivotaltracker.com/story/show/153728040)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,18 @@
 # Twilio API credentials
-export TWILIO_ACCOUNT_SID=AC807f0beda34c3b254e0cf785a42420af
-export TWILIO_AUTH_TOKEN=4de3c67a78763ae5c4e67a2cadce45f9
+export TWILIO_ACCOUNT_SID=AC1ead505ca3ff85324c9c26dbceed3296
```
Comment (cyberomin) [inline .env]: Can you please remove this and every other secret here and use an environment variable instead. Thanks.

**Comment by Thegaijin** [comment]
@cyberomin Done.

**Review by kaguna — COMMENTED** [review]
LGTM

**Comment by mwaz** [comment]
LGTM

**Review by mwaz — COMMENTED** [review]
👍 LGTM

**Review by jimmykimani — APPROVED** [review]

**Pull request merged**